our-process/policies.md

```diff
@@ -22,31 +22,34 @@ When submitting a package, please make sure that your GitHub notification settin
 
 ## Submitting your package for review in other venues
 
-We strongly suggest submitting your package for review with pyOpenSci before:
+We recommend submitting your package for review with pyOpenSci before:
 
-*  publishing on `PyPI` or `conda`;
-*  submitting a software paper describing the package to a journal.
+- publishing to a package repository, such as `PyPI` and `conda-forge`;
+- submitting a software paper describing the package to a journal.
 
 Review feedback may result in major improvements and updates to your package,
 including changes that could be break package functionality.
 
-We do not consider previous publication on `PyPI` or `conda` or in other venues
+We do not consider previous publication on `PyPI` or `conda-forge` or in other venues
```

Comment:
Maybe reframe slightly to:
By applying reviewer or editor recommendations, even previously published packages on PyPI, conda-forge, or in other venues can improve their user's experience with future version of the package.

Reply:
excellent. i suspect this might be more relevant to R / cran (our language started with language in the rOpenSci guide! i've reworded as follows:

> Applying reviewer or editor recommendations to your package can improve your user's experience with future versions of your package 
> even if your package is already published on `PyPI` or `conda-forge`.